tests/monitoring/monitoring.go

```diff
@@ -162,6 +165,48 @@ var _ = Describe("[sig-monitoring]Monitoring", Serial, decorators.SigMonitoring,
 		})
 	})
 
+	Context("VM Memory Alerts", func() {
+		BeforeEach(func() {
+			libmonitoring.ReduceAlertPendingTime(virtClient)
+		})
+
+		It("KubeVirtVMGuestMemoryAvailableLow should fire without swap under sustained memory pressure", func() {
+			By("Starting a Fedora VMI with low memory and no swap")
+			vmi := libvmifact.NewFedora(libnet.WithMasqueradeNetworking(), libvmi.WithMemoryRequest("256Mi"))
+			vmi = libvmops.RunVMIAndExpectLaunch(vmi, libvmops.StartupTimeoutSecondsHuge)
+			Expect(console.LoginToFedora(vmi)).To(Succeed())
+
+			By("Disabling swap and allocating most of the memory to keep usable very low")
+			Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
+				&expect.BSnd{S: "sudo swapoff -a || true\n"},
+				&expect.BExp{R: console.PromptExpression},
+				&expect.BSnd{S: "mem_k=$(awk '/MemTotal/{print int($2*95/100)}' /proc/meminfo); sudo stress-ng --vm 1 --vm-bytes ${mem_k}k --vm-keep &\n"},
+				&expect.BExp{R: console.PromptExpression},
+			}, 30)).To(Succeed())
+
+			By("Waiting for the low available memory alert")
+			libmonitoring.VerifyAlertExistWithCustomTime(virtClient, "KubeVirtVMGuestMemoryAvailableLow", 5*time.Minute)
```

Comment:
use `libmonitoring.VerifyAlertExist` like other tests it's calling `VerifyAlertExistWithCustomTime` with 120 seconds. 5 minutes is too much also not make sense since using `libmonitoring.ReduceAlertPendingTime(virtClient)`